### PR TITLE
DDF-3704 Reorder installer pages

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Reorder installer pages to Welcome -> Setup Type -> Claims -> Settings -> Finish
#### Who is reviewing it? 
@kcover @peterhuffer @emmberk 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams/ui
#### Choose 2 committers to review/merge the PR. 
@andrewkfiedler
@shaundmorris
@stustison
#### How should this be tested? (List steps with links to updated documentation)
Install DDF and verify there are no issues going through the installer.

#### What are the relevant tickets?
[DDF-3704](https://codice.atlassian.net/browse/DDF-3704)

### CODE REVIEW

File: ui/packages/ui/src/main/webapp/js/views/installer/InstallerMain.view.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ define([
         },
         changePage: function() {
             //close whatever view is open
-            var welcomeStep = 0, guestClaimsStep = 1, profileStep = 2, configStep = 3,  finishStep = 4;
+            var welcomeStep = 0, guestClaimsStep = 2, profileStep = 1, configStep = 3,  finishStep = 4;
```
Comment (emmberk) [inline ui/packages/ui/src/main/webapp/js/views/installer/InstallerMain.view.js]: Is there any documentation that needs to be updated?

Comment (clockard) [reply ui/packages/ui/src/main/webapp/js/views/installer/InstallerMain.view.js]: I didn't find any documentation that specified the order which is the only thing that changed.

Comment (clockard) [reply ui/packages/ui/src/main/webapp/js/views/installer/InstallerMain.view.js]: Looking again I think I might need to change the `:order:` number in the headers so they get added in the same order they appear in the installer.

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Pull request merged**